### DISCUSSION TITLE
End him rightly

### DISCUSSION THREAD

**Discussion by Lambosaurus** [Lambosaurus/hives-and-torches#35] [Feats]
Unscrew your pommel, and throw it at your foe.

Make a finesse roll. On success, you may attack the foe with `2x` advantage.
All subsequent attacks are made with a disadvantage

**Comment by LouderthanThunder** [comment]
For fuck sake 😂

**Reply by Lambosaurus** [reply]
🦕🦕🦕🦕🦕🦕🦕🦕🦕🦕🦕🦕🦕
